README.md

```diff
@@ -163,3 +163,23 @@ defined by a composition of a translation and a linear transformation. An
 (and controllable) if you construct it from a composition of a linear map
 and a translation, e.g. `Translation(v) ∘ LinearMap(v)` (or any combination of
 `LinearMap`, `Translation` and `AffineMap`).
+
+#### Perspective transformations
+
+The perspective transformation maps real-space coordinates to those on a virtual
+"screen" of one lesser dimension. For instance, this process is used to render
+3D scenes to 2D images in computer generated graphics and games. It is an ideal
+model of how a pinhole camera operates and is a good approximation of the modern
+photography process.
+
+The `PerspectiveMap()` command creates a singleton `Transformation` to perform
+the projective mapping. It can be applied individually, but is particularly
+powerful when composed with an `AffineMap` containing the position and
+orientation of the camera in your scene. For example, to transfer `points` in 3D
+space to 2D `screen_points` giving their projected locations on a virtual camera
+image, you might use the following code:
+
+```julia
+cam_transform = PerspectiveMap() ∘ inv(AffineMap(cam_rotation, cam_position))
```

Comment:
> I think it would bad to return to the days of thinking about all these problems as matrix problems

Agreed, to me it seems that the matrix version with homogeneous coordinates was always a bit of a hack, and ultimately restricts you to a certain subclass of blessed transformations.  It also reduces efficiency, unless you go to great lengths to flag out all the special cases... for example:

https://github.com/qt/qtbase/blob/6bceb4a8a9292ce9f062a38d6fe143460b54370e/src/gui/math3d/qmatrix4x4.h#L744

(This kind of code isn't at all unique to Qt.  It's the same in the java standard library and many other places which decided to bless 4x4 matrices as a definitive kind of transformation.)

A transformation chain with each part expressed as a type is pretty great, and in principle has slightly superiour performance.  (I probably wouldn't bother in C++ due to the amount of template pain it would cause, but in julia it's not a syntactical or design hurdle :-) )